### PR TITLE
 Step: 2.3.0_SNPASHOT -> 2.4.0

### PR DESCRIPTION
**Pull request by AlvaroVega**
Mimics PR https://github.com/telefonicaid/fiware-cygnus/pull/1896

### CODE REVIEW

File: cygnus-common/pom.xml [hunk 1]
```diff
@@ -5,7 +5,7 @@
```
Comment (fgalan) [inline cygnus-common/pom.xml:5]: Looking to https://github.com/telefonicaid/fiware-cygnus/pull/1896/files, CNR file should be flushed in the release closing PR.

`rm CHANGES_NEXT_RELEASE ; touch CHANGES_NEXT_RELEASE` does the trick :)

File: cygnus-ngsi/spec/SPECS/cygnus-ngsi.spec [hunk 2]
```diff
@@ -99,6 +99,15 @@ rm -rf $RPM_BUILD_ROOT
...
 
 %changelog
+* Fri Sep 4 2020 Alvaro Vega <alvaro.vegagarcia@telefonica.com> 2.4.0
+- [cygnus-ngsi][RowAggregator] Handle metadata as json. This is to upgrade json-like sinks behaviour. (#1902)
+- [cygnus-ngsi][ColumnAggregator] Fix attribute type key to make it unique. (#1904)
+- [cygnus-ngsi][PostgisSink, PostgreSQLSink] Implement records expiration for Postgis and PostgreSQL sinks (#1915)
+- [cygnus-ngsi][ColumnAggregator] Handle metadata as json. This is to upgrade json-like sinks behaviour. (#1902)
+- [cygnus-ngsi][CKANSink] Fix bug about bad service name used by CKAN orgName with a name mapping (#1910)
+- [cygnus-ngsi][CKANSink] Add flag option to disable metatada in CKAN (#1912)
+- [cygnus-common][SQLBackendImpl] Exception handle for connection/statement SQL objects (#1907)
```
Comment (fgalan) [inline cygnus-ngsi/spec/SPECS/cygnus-ngsi.spec:109]: This line should me moved to cygnus-commons .spec

File: cygnus-ngsi/spec/SPECS/cygnus-ngsi.spec [hunk 3]
```diff
@@ -99,6 +99,15 @@ rm -rf $RPM_BUILD_ROOT
 %{_project_install_dir}
 
 %changelog
+* Fri Sep 4 2020 Alvaro Vega <alvaro.vegagarcia@telefonica.com> 2.4.0
+- [cygnus-ngsi][RowAggregator] Handle metadata as json. This is to upgrade json-like sinks behaviour. (#1902)
+- [cygnus-ngsi][ColumnAggregator] Fix attribute type key to make it unique. (#1904)
+- [cygnus-ngsi][PostgisSink, PostgreSQLSink] Implement records expiration for Postgis and PostgreSQL sinks (#1915)
+- [cygnus-ngsi][ColumnAggregator] Handle metadata as json. This is to upgrade json-like sinks behaviour. (#1902)
+- [cygnus-ngsi][CKANSink] Fix bug about bad service name used by CKAN orgName with a name mapping (#1910)
+- [cygnus-ngsi][CKANSink] Add flag option to disable metatada in CKAN (#1912)
```
Comment (fgalan) [inline cygnus-ngsi/spec/SPECS/cygnus-ngsi.spec:108]: For the sake of homogenity:

```suggestion
- [cygnus-ngsi][RowAggregator] Handle metadata as json. This is to upgrade json-like sinks behaviour (#1902)
- [cygnus-ngsi][ColumnAggregator] Fix attribute type key to make it unique (#1904)
- [cygnus-ngsi][PostgisSink, PostgreSQLSink] Implement records expiration for Postgis and PostgreSQL sinks (#1915)
- [cygnus-ngsi][ColumnAggregator] Handle metadata as json. This is to upgrade json-like sinks behaviour (#1902)
- [cygnus-ngsi][CKANSink] Fix bug about bad service name used by CKAN orgName with a name mapping (#1910)
- [cygnus-ngsi][CKANSink] Add flag option to disable metatada in CKAN (#1912)
```

File: cygnus-twitter/spec/SPECS/cygnus-twitter.spec [hunk 4]
```diff
@@ -99,6 +99,8 @@ rm -rf $RPM_BUILD_ROOT
 %{_project_install_dir}
 
 %changelog
+* Fri Sep 4 2020 Alvaro Vega <alvaro.vegagarcia@telefonica.com> 2.4.0
```
Comment (fgalan) [inline cygnus-twitter/spec/SPECS/cygnus-twitter.spec:102]:
```suggestion
* Fri Sep 04 2020 Alvaro Vega <alvaro.vegagarcia@telefonica.com> 2.4.0
```

(Not sure but in older entries we did that way...)

File: cygnus-ngsi/spec/SPECS/cygnus-ngsi.spec [hunk 5]
```diff
@@ -99,6 +99,15 @@ rm -rf $RPM_BUILD_ROOT
 %{_project_install_dir}
 
 %changelog
+* Fri Sep 4 2020 Alvaro Vega <alvaro.vegagarcia@telefonica.com> 2.4.0
```
Comment (fgalan) [inline cygnus-ngsi/spec/SPECS/cygnus-ngsi.spec:102]:
```suggestion
* Fri Sep 04 2020 Alvaro Vega <alvaro.vegagarcia@telefonica.com> 2.4.0
```

(Not sure but in older entries we did that way...)

File: cygnus-common/spec/SPECS/cygnus-common.spec [hunk 6]
```diff
@@ -171,6 +171,8 @@ rm -rf $RPM_BUILD_ROOT
 /var/run/%{_project_name}
 
 %changelog
+* Fri Sep 4 2020 Alvaro Vega <alvaro.vegagarcia@telefonica.com> 2.4.0
```
Comment (fgalan) [inline cygnus-common/spec/SPECS/cygnus-common.spec:174]:
```suggestion
* Fri Sep 04 2020 Alvaro Vega <alvaro.vegagarcia@telefonica.com> 2.4.0
```

(Not sure but in older entries we did that way...)

File: cygnus-ngsi/spec/SPECS/cygnus-ngsi.spec (hunk 2 above)
Comment (AlvaroVega) [reply cygnus-ngsi/spec/SPECS/cygnus-ngsi.spec]: That's right! Fixed in 9d167ab5276e8dc6f157493045a073576a87d92b

File: cygnus-common/pom.xml (hunk 1 above)
Comment (AlvaroVega) [reply cygnus-common/pom.xml:5]: fixed in be3d8a35bc206ccf4b1dcab2d3721bab6578ab80

**Review by fgalan — APPROVED** [review]
LGTM

**Pull request merged**